### ISSUE TITLE
Ancient Focus Broken

### ISSUE THREAD

**Issue by linkbeta5** [TheCodex6824/ThaumicAugmentation#336]
It should be noted that I am playing the 1.12.2 pack v1.5.4

I'm trying to use the ancient focus with a foci manipulator, however all it does is take the Vis Crystals needed, drains the Vis required from the air, and than does nothing. The animation where the crystals are circling around the focus continues to play, and the focus remains incomplete. 
![image](https://user-images.githubusercontent.com/125415804/235268226-574b1bef-1c32-4ddf-ad1e-0236faad161f.png)
![image](https://user-images.githubusercontent.com/125415804/235268241-c7f4703f-2757-492a-baf1-84b68b2c9478.png)

**Comment by TheCodex6824** [comment]
As far as I know, this bug is an issue that can possibly occur with all foci in the focal manipulator - I haven't been able to figure out why it exactly happens because I can never seem to get it to trigger for me. If you can make this happen consistently, please share the setup to do so - it would be nice to finally make a fix for this.

**Comment by linkbeta5** [comment]
Yes, I can very much make this happen consistently with Ancient Focuses. I can get other Focuses to work (ones that are NOT involved with Thaumic Augmentation).
So what exactly do I need to share "setup" wise?

**Comment by TechnoMysterio** [comment]
If you could list what blocks/devices/items you're using, the steps you take to (not) create a bugged focus, other details like what spell you're making and how much Vis/XP it takes, basically anything to ensure that the bug can be consistently replicated so the problem can be isolated and fixed.
